simulation/src/test/resources/report.md

```diff
@@ -11,26 +11,26 @@
                live_reloading[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=97.2%	client_mean=PT4.8253936S   	server_cpu=PT2H46.79S     	client_received=2500/2500	server_resps=2500	codes={200=2430, 500=70}
-                   live_reloading[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=90.3%	client_mean=PT3.7919392S   	server_cpu=PT1H55M28.37S  	client_received=2500/2500	server_resps=2500	codes={200=2257, 500=243}
-                             live_reloading[UNLIMITED_ROUND_ROBIN].txt:	success=68.1%	client_mean=PT2.845456S    	server_cpu=PT1H58M33.64S  	client_received=2500/2500	server_resps=2500	codes={200=1703, 500=797}
+                   live_reloading[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=90.9%	client_mean=PT3.746748S    	server_cpu=PT1H55M56.12S  	client_received=2500/2500	server_resps=2500	codes={200=2273, 500=227}
+                             live_reloading[UNLIMITED_ROUND_ROBIN].txt:	success=88.8%	client_mean=PT2.807144S    	server_cpu=PT1H56M57.86S  	client_received=2500/2500	server_resps=2500	codes={200=2221, 500=279}
                 one_big_spike[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=100.0%	client_mean=PT1.135007332S 	server_cpu=PT2M49.65S     	client_received=1000/1000	server_resps=1131	codes={200=1000}
                     one_big_spike[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.737618067S 	server_cpu=PT2M44.55S     	client_received=1000/1000	server_resps=1097	codes={200=1000}
-                              one_big_spike[UNLIMITED_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT1.21781793S  	server_cpu=PT8M23.55S     	client_received=1000/1000	server_resps=3357	codes={200=1000}
+                              one_big_spike[UNLIMITED_ROUND_ROBIN].txt:	success=99.9%	client_mean=PT1.000523583S 	server_cpu=PT7M37.35S     	client_received=1000/1000	server_resps=3049	codes={200=999, 429=1}
 one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=64.4%	client_mean=PT2.6866096S   	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1611, 500=889}
- one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=65.4%	client_mean=PT0.9852512S   	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1636, 500=864}
-           one_endpoint_dies_on_each_server[UNLIMITED_ROUND_ROBIN].txt:	success=65.9%	client_mean=PT0.6S         	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1648, 500=852}
+ one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=65.0%	client_mean=PT1.1287088S   	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1625, 500=875}
+           one_endpoint_dies_on_each_server[UNLIMITED_ROUND_ROBIN].txt:	success=64.6%	client_mean=PT0.6S         	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1616, 500=884}
```

Comment:
slightly lower, but the one-endpoint-on-each-server results aren't something we optimize for